Sources/Recycling/ListViews/AdManagement/Cells/UserAdManagementStatisticsCell.swift

```diff
@@ -3,15 +3,28 @@
 //
 
 public class UserAdManagementStatisticsCell: UITableViewCell {
+
+    // MARK: - Public
+
     public var itemModels = [StatisticsItemModel]() {
         didSet { updateStackViewContent() }
     }
+
+    // MARK: - Private
+
+    private lazy var emptyView: StatisticsItemEmptyView = {
+        let view = StatisticsItemEmptyView(title: "Følg med på effekten", description: "Etter at du har publisert annonsen din kan du se statistikk for hvor mange som har sett annonsen din, favorisert den og som har fått tips om den.")
```

Comment:
Should we have inline texts like this? I don't think it necessarily makes sense to inject "static" texts like these from the main app, but I also think it doesn't look right to have them hard-coded inline.

Would it make sense to have a `Localizable.strings` inside FinniversKit?

Reply:
This isn't correct, brain fart.

The text is there for demo purposes, but in the actual app they will be in the `Localizable.strings` file.

I'll remove them from the constructor and make them part of the model or a separate model.